### ISSUE TITLE
Feature Request : Add check for ground truth label range in binary_crossentropy

### ISSUE THREAD

**Issue by mrgloom** [keras-team/keras#14046]
```
python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"
v1.14.0-rc1-22-gaf24dc91b5 1.14.0

python -c 'import keras as k; print(k.__version__)'
Using TensorFlow backend.
2.1.6

```
`y_true` values are out of the valid range in this example:
```
    y_pred = np.array([0.4, 0.6])
    y_pred = np.expand_dims(y_pred, axis=0)
    y_true = np.array([-2.0, 2.0])
    y_true = np.expand_dims(y_true, axis=0)

    loss = keras.losses.binary_crossentropy(
        K.variable(y_true),
        K.variable(y_pred)
    )

    print("K.eval(loss):", K.eval(loss))
```

**Comment by old-school-kid** [comment]
This issue can be closed. Please refer to the discussion in this [PR](https://github.com/keras-team/keras/pull/15744)

**Comment by Suzan009** [comment]
Possible issue for this PR. https://github.com/keras-team/keras/pull/15744#issuecomment-1003574394

**Comment by sushreebarsa** [comment]
@mrgloom Can we  move this issue to closed status if it is resolved as per the above comments ?Please let us know.Thank you!
